usaspending_api/broker/lookups.py

```diff
@@ -7,6 +7,7 @@
 
 LookupType = namedtuple("LookupType", ["id", "name", "desc"])
 
+# External Data Load Date
```

Comment:
What's the purpose of this inline code comment? This comment leaves me as a brief viewer of the code my confused after reading it.

Reply:
Just separating the code handling `external_data_type` data vs. `delta_table_load_version` lookups.